tests/helm/testdata/goldenfile/opentelemetry_operator_instrumentation_cr_job/job.output.yaml

```diff
@@ -1,66 +1,37 @@
 ---
-# Source: sumologic/templates/opentelemetry-operator/job-instrumentation-cr.yaml
```

Comment:
Why does anything change in this file?

Reply:
@swiatekm-sumo I have reverted unwanted changes added by local output file regenerate. Didn't noticed these automatic updates from local make regenerate.